addons/mail/static/src/core/common/message_reaction_button.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.MessageReactionButton">
-        <button class="btn px-1 py-0 lh-1 rounded-0" tabindex="1" title="Add a Reaction" aria-label="Add a Reaction" t-ref="emoji-picker"><i class="oi fa-lg oi-smile-add"/></button>
+        <button class="btn px-1 py-0 lh-1 rounded-0 rounded-start-1" tabindex="1" title="Add a Reaction" aria-label="Add a Reaction" t-ref="emoji-picker"><i class="oi fa-lg oi-smile-add"/></button>
```

Comment:
That's not necessarily the 1st message action.
In theory you should pass classname to 1st message action, so that this is a conditional classname.